resources/dicts/patrols/plains/training/any.json

```diff
@@ -2828,5 +2828,125 @@
+        "decline_text": "It wouldn't help to talk to app1 while {PRONOUN/app1/subject} {VERB/app1/is/are} in this state. p_l tells {PRONOUN/app1/poss} {PRONOUN/p_l/subject}'ll leave {PRONOUN/app1/poss} alone, and to come find {PRONOUN/p_l/poss} if {PRONOUN/app1/subject}'d like to keep training today. app1 never does.",
+        "success_outcomes": [
+            {
+                "text": "p_l waits patiently for app1 to calm down. Once {PRONOUN/app1/subject} can listen, p_l tells {PRONOUN/app1/poss} a secret: no warrior has ever felt like a warrior. Not even p_l. You will always feel like you, p_l says, but don't be afraid. You <i>must</i> be you before you are a warrior. That is what c_n truly needs. You. ",
+                "exp": 10,
+                "stat_trait": ["loving","compassionate","wise","calm","thoughtful","faithful","loyal","sincere","righteous"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "App1 sniffles alone for a few moments before facing {PRONOUN/app1/poss} mentor. App1's eyes widen at p_l standing over {PRONOUN/app1/poss}, claws unsheathed and fur bristled. <i>That</i> is weakness. c_n, and more importantly, p_l will not tolerate weakness. Get back into training, app1, and do not let p_l see you act in this way again.",
+                "exp": 10,
+                "stat_trait": ["bloodthirsty","vengeful","arrogant","troublesome"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["dislike","respect"],
+                        "amount": 10
+                    },
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["trust","platonic","comfort"],
+                        "amount": -10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l's whiskers twitch in annoyance at this childish outburst. If app1 doesn't <i>want</i> to be a warrior, that's fine by p_l. {PRONOUN/p_l/subject}'ll be in the training grounds, waiting to train the greatest warrior c_n's ever seen. {PRONOUN/p_l/subject} {VERB/p_l/walk/walks} away from app1, and a hidden smile dashes across {PRONOUN/p_l/poss} muzzle as {PRONOUN/p_l/subject} {VERB/p_l/hear/hears} app1 start to chase after {PRONOUN/p_l/poss}.",
```

Comment:
also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l

I love this angle for these personalities!